content/en/account_management/billing/usage_attribution.md

```diff
@@ -137,4 +136,5 @@ Each color block represents a unique tag value for each tag.
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: https://docs.datadoghq.com/api/v1/usage-metering/#get-monthly-usage-attribution
+[2]: https://docs.datadoghq.com/api/v1/usage-metering/#get-hourly-usage-attribution
```

Comment:
hey @StefonSimmons it looks like the `format_link` precommit hook caused this change.   we need to be sure these changing the shortcode delimiters won't introduce an issue with links being parsed improperly.   its working on this test page but it would be good to do some tests on pages with more `site-region` shortcodes, maybe also within `tabs` etc.